spec/takeaway_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'spec_helper.rb'
+require_relative '../lib/takeaway.rb'
+
+describe Takeaway do
+
+   it 'returns a properly formatted list of dishes available when asked' do
+    expect(subject.see).to eq '44 : Duck a la banana : £4.50, 10 : Deep fried Mars bar : £0.10, 20 : Birdseye potato waffles : £1.99, 69 : Trio! Trio. : £1.50'
+  end
```

Comment:
Lint/BlockAlignment: `end` at 8, 2 is not aligned with `it 'returns a properly formatted list of dishes available when asked' do` at 6, 3